src/gui/systray.cpp

```diff
@@ -105,6 +105,13 @@ void Systray::create()
     }
     hideWindow();
     emit activated(QSystemTrayIcon::ActivationReason::Unknown);
+
+    const auto folderMap = FolderMan::instance()->map();
+    for (const auto *folder : folderMap) {
+        if(!folder->syncPaused()) {
```

Comment:
Somehow it wasn't here before or I spot it only now: please add a space after if